monailabel/endpoints/datastore.py

```diff
@@ -183,3 +183,15 @@ async def api_download_image(image: str):
 @router.get("/label", summary="Download Label")
 async def api_download_label(label: str, tag: str):
     return download_label(label, tag)
+
+
+@router.put("/updatelabelinfo", summary="Update label info")
+async def api_update_label_info(image: str, params: str = Form("{}")):
+    return update_label_info(image, params)
+
+
+def update_label_info(id: str, params: str = Form("{}")):
+    save_params: Dict[str, Any] = json.loads(params) if params else {}
+    instance: MONAILabelApp = app_instance()
+    instance.datastore().update_image_info(id, save_params)
```

Comment:
I agree that we could have used "instance.datastore().update_label_info()".
However, in our use case, we thought of the idea of pre-tagging the images based on image quality or segmentation difficulty before actually starting the segmentation. Therefore, we wanted to match this additional information to the appropriate image.

However, if you want me to adapt this code accordingly. please let me know.

Reply:
update_label_info => datastore().update_label_info()
update_image_info => datastore().update_image_info()

lets go with this.. otherwise it shall be confusing.. endpoint vs datastore function mapping..

Reply:
Done